src/gcp/cloudfunctions.js

```diff
@@ -20,7 +20,7 @@ function _functionsOpLogReject(func, type, err) {
   } else {
     logger.info(err.message);
   }
-  return Promise.reject(err.message);
+  return Promise.reject(func);
```

Comment:
I doubt `func` is what is wanted here. This probably should just be reverted? Or actually changed to `err`? If this is a rejection handler that isn't doing anything else, it makes sense for this to re-throw the `err` again.

Reply:
We use the value from this in the other file to get the name of the function that failed, which is `func`, so it should be returned in some form. Will reject with both the name and error.